guides/v2.2/cloud/before/before-workspace-magento-prereqs.md

```diff
@@ -46,7 +46,8 @@ Although {{ site.data.var.ee }} supports the Apache web server, {{ site.data.var
 
 ## PHP (local) {#php}
 
-Install [PHP](https://glossary.magento.com/php) on your local. We recommend PHP 7.0. For information on installing PHP, see these instructions for [CentOS]({{ page.baseurl }}/install-gde/prereq/php-centos.html) and [Ubuntu]({{ page.baseurl }}/install-gde/prereq/php-ubuntu.html). For instructions for another OS, see the [PHP documentation](http://php.net/manual/en/install.php).
+Install [PHP](https://glossary.magento.com/php) on your local. We recommend PHP 7.2.
```

Comment:
Cloud works with a Magento app template. For certain Magento 2.2, you cannot use PHP 7.2. I'd rather you delete the:
```suggestion
Install [PHP](https://glossary.magento.com/php) on your local.
```